feat: fb-ROOT-227: Add Google Analytics 4 measurement ID

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/19579373094)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/674
<!-- Follow Merge -->

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19372012226

matt-bernstein commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/19579378339)
> Successfully merged:  145 files changed, 9266 insertions(+), 3681 deletions(-)

matt-bernstein committed:
Merge branch 'develop' into 'fb-ROOT-227'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19579378339

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19579373094
